src/app/settings/settings.component.html

```diff
@@ -78,6 +78,19 @@ <h5 class="no-margin">{{ 'FIRST_DAY_OF_WEEK' | translate }}</h5>
         </mdl-select>
       </div>
       <div class="settings-control">
+        <h5 class="no-margin">{{ 'TIME_FORMAT' | translate }}</h5>
+        <mdl-select
+          [ngModel]="timeFormat"
+          [disabled]="updatingTimeFormat"
+          (change)="changeTimeFormat($event)"
+        >
+          <mdl-option
+            *ngFor="let format of timeFormats"
+            [value]="TimeFormat[format]"
```

Comment:
If AUTO is set, selected value is not visible
<img width="362" alt="screen shot 2017-07-16 at 16 20 52" src="https://user-images.githubusercontent.com/2853979/28246376-c87c6d58-6a42-11e7-9b1e-07bfd0493177.png">